lab-teddy/.env

```diff
@@ -0,0 +1,3 @@
+APP_SECRET='dau-boot'
```

Comment:
Definitely get in the habit of ensuring you have a gitignore file that hides your env, db, and node modules. I would do this in your setup stage.